### DISCUSSION TITLE
Relocate only finds a portion

### DISCUSSION THREAD

**Discussion by designersocks** [rshea0/enjinn#98] [Help]
Hi there,

Thanks a million for this great application. I'm trying to use relocate but it only finds a portion of the missing tracks, while they are all in the drive I selected. Also, it doesn't seem to update the missing tracks. When I run 'find missing' again, it just shows the same tracks again.

**Comment by shayded-exe** [comment]
Hey, sorry but I'm no longer able to support this app due to other priorities. I'm open to PRs if you or anyone else wants to fix/improve it.

I'm also planning to sell my Prime 4 for something not Denon. The standalone-ability just isn't worth the issues and missing features.

I still need to push an update to remove the licensing and update the readme. I've already made it free on gumroad (you can have a refund if you want).
